packages/react/src/PageHeader/PageHeader.tsx

```diff
@@ -176,6 +183,7 @@ const Root = React.forwardRef<HTMLDivElement, React.PropsWithChildren<PageHeader
         ref={rootRef}
         as={as}
         className={clsx(enabled && classes.PageHeader, className)}
+        data-has-border={hasBorder}
```

Comment:
```suggestion
        data-has-border={hasBorder ? '' : undefined}
```

Small suggestion so that in the HTML it shows up as `data-has-border` when true and then we can target it with just `[data-has-border]`. When it's not in the DOM it's removed.

I also asked this question in our CSS Modules group to see if there was a style preference here 🤔 Not sure what folks prefer and this is definitely just a style thing